go/porcelain/deploy_linux_test.go

```
@@ -0,0 +1,399 @@
+		sort.Strings(labels)
+		var b strings.Builder
+		for _, k := range labels {
+			fmt.Fprintf(&b, "\n  - %s (x%d)", k, fileswaps[k])
+		}
+		t.Fatalf("DoDeploy: contents changed during deploy: %d branch/window combination(s):%s",
+			len(labels), b.String())
+	}
+}
+
+func TestDoDeploy_SymlinkAncestorTOCTOURace(t *testing.T) {
+	if runtime.GOOS == "windows" {
```

golangci-lint (2.12.2)

[error] SA4032: due to the file's build constraints, runtime.GOOS will never equal "windows"

(staticcheck)